cmake/modules/RootNewMacros.cmake

```diff
@@ -987,6 +990,11 @@ function(ROOT_STANDARD_LIBRARY_PACKAGE libname)
                        )
   endif(ARG_OBJECT_LIBRARY)
 
+  # Dictionary might include things from the current src dir. Alas there is no
+  # way to set the include directory for a source file (except for th generic
+  # COMPILE_FLAGS), so this needs to be glued to the target.
+  target_include_directories(${libname} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
This is e.g. for tests - I've updated the comment. And the dictionary compilation does not add an implied `-I${CMAKE_CURRENT_SOURCE_DIR}` without the change above, because the dictionary is in the build directory.

Reply:
With `CMAKE_INCLUDE_CURRENT_DIR` even `rootcling` will probably have the current directory in the list of includes, which means we can probably remove this.